src/cryptography/x509/extensions.py

```diff
@@ -188,8 +189,21 @@ def from_issuer_public_key(cls, public_key):
 
     @classmethod
     def from_issuer_subject_key_identifier(cls, ski):
+        try:
```

Comment:
Please use explicit `isinstance` rather than catching `AttributeError`

Reply:
That won't capture the set of all possible bullshit people may have done to make this work. Most users who saw this are unlikely to realize "oh it expects an Extension, which has a `value`". I think there's going to be code out there that just creates a wrapper object like 

```
class Nonsense:
    def __init__(self, value):
        self.value = value
````
and pass that in.